frontend/src/js/components/PageViewer/FindInput.tsx

```diff
@@ -1,42 +1,47 @@
-import _ from "lodash";
+import _, { uniq } from 'lodash';
 import React, {
   FC,
   KeyboardEventHandler,
   useCallback,
-  useEffect,
+  useEffect, useMemo,
   useState,
-} from "react";
+} from 'react';
 import DownIcon from "react-icons/lib/md/arrow-downward";
 import UpIcon from "react-icons/lib/md/arrow-upward";
 import styles from "./FindInput.module.css";
+import { HighlightForSearchNavigation } from './model';
+import { Loader } from 'semantic-ui-react';
 
 type FindInputProps = {
   value: string;
   setValue: (v: string) => void;
   performFind: (query: string) => Promise<void>;
+  isPending: boolean;
   jumpToNextFindHit: () => void;
   jumpToPreviousFindHit: () => void;
-  hits: number[];
-  lastPageHit: number;
+  // TODO: could be null?
```

Comment:
is this simple enough to check before merging?

Reply:
it can't be null actually, it's state that initialised to `[]` in `PageViewer.tsx`. will delete comment